lib/ParseSIL/ParseSIL.cpp

```diff
@@ -997,12 +998,10 @@ static bool parseSymbolicValue(SymbolicValue &value, SILParser &SP,
   }
   // Handle string literals.
   if (P.Tok.is(tok::string_literal)) {
-    // TODO: Uncomment when `getStringValue` is implemented.
-    // StringRef rawString = P.Tok.getText().drop_front().drop_back();
-    // value = SymbolicValue::getStringValue(rawString, allocator);
-    // return false;
-    P.diagnose(P.Tok, diag::sil_graph_op_unhandled_attr_value);
-    return true;
+    StringRef rawString = P.Tok.getText().drop_front().drop_back();
+    value = SymbolicValue::getString(rawString, allocator);
+    P.consumeToken();
```

Comment:
Please pass tok::string_literal to consumeToken (to follow convention)

Reply:
Done.